dist/build.js

```diff
@@ -1626,6 +1629,15 @@ utils.getBooleanOrNull = function(a) {
 utils.delay = function(a, b) {
   isNaN(b) || 0 >= b ? a() : setTimeout(a, b);
 };
+utils.getClientHints = function() {
```

Comment:
can you add a comment on why we are doing this for reference?

Reply:
build.js is a generated file by google closure compiler which removes all the comments.
The original code file has a comment about the function
https://github.com/BranchMetrics/web-branch-deep-linking-attribution/blob/96550a89fff6e93f1fdbb53edf8e6f3a4796e299/src/1_utils.js#L1276